build.zig

```diff
@@ -43,15 +43,16 @@ fn setupTesting(b: *std.Build, target: std.zig.CrossTarget, optimize: std.builti
     const test_step = b.step("test", "Run unit tests");
     test_step.dependOn(&run_unit_tests.step);
 
-    const test_dirs = [_][]const u8{"util"};
+    const test_dirs = [_][]const u8{ "util", "." };
     for (test_dirs) |dir| {
         addTestsFromDir(b, test_step, dir, target, optimize);
     }
 }
 
 fn addTestsFromDir(b: *std.Build, test_step: *std.Build.Step, dir_path: []const u8, target: std.zig.CrossTarget, optimize: std.builtin.OptimizeMode) void {
-    const iterableDir = std.fs.cwd().openIterableDir(dir_path, .{}) catch {
-        std.debug.print("Failed to open directory: {any}\n", .{dir_path});
+    const iterableDir = std.fs.cwd().openDir(dir_path, .{ .iterate = true }) catch {
```

Comment:
one example of a Zig version incompatibility... More to come in the latest master ;-)